magento/pwa-studio
# PWA-622: Update Venia's button

## Description
Adjust `Button` styles to match the new design

## Related Issue
https://jira.corp.magento.com/browse/PWA-622

## Acceptance 
- Verify all buttons are updated throughout the app
- Ensure there is no broken functionality with the styling change

### Verification Stakeholders
@jimbo 
@dhaecker 
@soumya-ashok 
@schensley 

### Specification



### Verification Steps
1. Navigate around the app following all known user flows
2. Verify all buttons previously present are present and work
3. Verify all buttons are positioned and readable

## Screenshots / Screen Captures (if appropriate)

## Checklist
